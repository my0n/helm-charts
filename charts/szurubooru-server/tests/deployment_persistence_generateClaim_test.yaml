suite: test deployment
templates:
  - configMap.yaml
  - deployment.yaml
tests:
  - it: should match snapshot
    chart:
      version: 1.0.0
      appVersion: 1.0.0
    set:
      persistence.enabled: true
    template: deployment.yaml
    asserts:
      - matchSnapshot: {}