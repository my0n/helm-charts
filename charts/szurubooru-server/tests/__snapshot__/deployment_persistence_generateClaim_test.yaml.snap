should match snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: szurubooru-server
        app.kubernetes.io/version: 1.0.0
        helm.sh/chart: szurubooru-server-1.0.0
      name: RELEASE-NAME-szurubooru-server
    spec:
      replicas: 1
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: szurubooru-server
      template:
        metadata:
          annotations:
            checksum/config: f59abc467dab25af2d84dcd44e0c5982fd5d648ce5595f0f9042b42c8e0de34f
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/name: szurubooru-server
        spec:
          containers:
          - env:
            - name: POSTGRES_DB
              value: szurubooru
            - name: POSTGRES_HOST
              value: ""
            - name: POSTGRES_PORT
              value: "5432"
            - name: LOG_SQL
              value: "0"
            image: szurubooru/server:1.0.0
            imagePullPolicy: IfNotPresent
            name: szurubooru-server
            ports:
            - containerPort: 6666
              name: api
              protocol: TCP
            resources: {}
            securityContext: {}
            volumeMounts:
            - mountPath: /.dockerenv
              name: dockerenv
              subPath: .dockerenv
            - mountPath: /data
              name: data
            - mountPath: /opt/app/config.yaml
              name: config-final
              subPath: config.yaml
          initContainers:
          - args:
            - /bin/sh
            - -c
            - touch /.dockerenv
            image: busybox
            imagePullPolicy: IfNotPresent
            name: szurubooru-server-generate-dockerenv
            volumeMounts:
            - mountPath: /.dockerenv
              name: dockerenv
              subPath: .dockerenv
          - args:
            - /bin/sh
            - -c
            - envsubst < /workdir/config.yaml > /processed/config.yaml
            env: null
            image: bhgedigital/envsubst:v1.0-alpine3.6
            imagePullPolicy: IfNotPresent
            name: szurubooru-server-init-config
            volumeMounts:
            - mountPath: /workdir
              name: config-template
            - mountPath: /processed
              name: config-final
          securityContext: {}
          serviceAccountName: RELEASE-NAME-szurubooru-server
          volumes:
          - configMap:
              name: RELEASE-NAME-szurubooru-server
            name: config-template
          - emptyDir: {}
            name: config-final
          - emptyDir: {}
            name: dockerenv
          - name: data
            persistentVolumeClaim:
              claimName: RELEASE-NAME-szurubooru-server
